helm_deploy/hmpps-welcome-people-into-prison-ui/Chart.yaml

```diff
@@ -5,8 +5,8 @@ name: hmpps-welcome-people-into-prison-ui
 version: 0.2.0
 dependencies:
   - name: generic-service
-    version: 1.5.0
+    version: 2.1.0
     repository: https://ministryofjustice.github.io/hmpps-helm-charts
   - name: generic-prometheus-alerts
-    version: 0.4.1
+    version: 1.1.0
```

Comment:
Removes a cronjob alerts that was deemed unfit, more details here https://github.com/ministryofjustice/hmpps-helm-charts/commit/58c301f701f1daa771a3406dafa7c0491397ed37